libs/minecraft/InventoryUtils.arucas

```diff
@@ -0,0 +1,308 @@
+import * from Minecraft;
+import * from minecraft.MinecraftUtils;
+import * from util.Collections;
+class InventoryUtil {
+    static var TOOLS = ['axe', 'shovel', 'pickaxe', 'hoe'];
+    static var debug = false;
+    static var selectedSlot = null;
+    static var preserveDurability = false;
+    static var pointer = 0;
+    static var hotbarSlotCache = {};
+    //class InventoryUtil does not offer static method, instead, it offers player-related options based on binded player entity
+    //Please never use setSeletedSlot outside! or, use sync() to refresh selected slot!
+    //This class is stable against server sync, will not suffer against server packets.
+    static fun sync() {
+        __docstr__ = 'Syncs selected slot and remove cached slot information';
```

Comment:
`__docstr__`?? This is not python
These are just slowing down these functions. Use comments!